pkg/p2p/libp2p/libp2p.go

```diff
@@ -171,6 +172,26 @@ func New(ctx context.Context, signer beecrypto.Signer, networkID uint64, overlay
 		return nil, err
 	}
 
+	cfg := rcmgr.InfiniteLimits
+
+	in, out := 10_000, 30_000
```

Comment:
These are quite large numbers. I assume empirical.

Probably init function at line 79 in this file should be removed:
```go
// nolint:gochecknoinits
func init() {
	goyamux.DefaultTransport.AcceptBacklog = 1024
	goyamux.DefaultTransport.MaxIncomingStreams = 5000
}
```